prime-router/src/main/kotlin/fhirengine/translation/hl7/schema/ConfigSchemaReader.kt

```diff
@@ -28,8 +30,27 @@ object ConfigSchemaReader : Logging {
         schemaClass: Class<out ConfigSchema<out ConfigSchemaElement>>
     ): ConfigSchema<*> {
         // Load a schema including any parent schemas.  Note that child schemas are loaded first and the parents last.
+        val schemaList = when (URI(schemaName).scheme) {
```

Comment:
Opted to fully fork here rather than attempting to support a mix of relative pathing and URIs since that proved to be very difficult to implement.

there is a ticket for switching everything over to URIs before any azure migration